### ISSUE TITLE
enable mlflow autolog for loss (default every 100 iterations)

### ISSUE THREAD

**Issue by Zethson** [mlf-core/mlf-core#157]
> If you want to log the loss or some other metric for every epoch, you have to set every_n_iter  parameter in the mlflow.tensorflow.autolog  function or in the mlflow.pytorch.autolog  function. The default is set to 100

Should quickly check whether the new mlflow.autolog() supports this and then likely go with a much smaller number.
Maybe even 1?

**Comment by KevinMenden** [comment]
I would go with 1, yes. It's not like we're saving a lot of data anyway (usually).

Think I already checked and it doesn't support it, but I will look into this soon and make a PR once ready.

**Comment by Zethson** [comment]
Think that your PR should wait for https://github.com/mlf-core/mlf-core/issues/130 to be resolved, which we're hoping to finalize this week.

Thanks!

**Comment by Zethson** [comment]
Goes very well with https://github.com/mlf-core/mlf-core/issues/237 and is trivial to do. Hence, I assigned you here as well, @Emiller88 .
